ReceiptsTests/ReceiptListViewControllerTests.swift

```diff
@@ -0,0 +1,19 @@
+import XCTest
+@testable import Receipts
+
+class ReceiptListViewControllerTests: XCTestCase {
+    
+    func testSyncActionAddsRowsToTableView() {
+        let storyboard = UIStoryboard(name: "Main", bundle: nil)
+        let listVC = storyboard.instantiateViewController(withIdentifier: "ReceiptListViewController") as! ReceiptListViewController
+        let storeURL = DeveloperSettings.randomReceiptStoreURL()
+        let store = ReceiptStore(fileURL: storeURL)
+        listVC.receiptStore = store
+        listVC.loadViewIfNeeded()
```

Comment:
`loadViewIfNeeded` is a helpful method to make sure the VC is properly loaded and ready to go for testing.